app/helpers/get_data.py

```diff
@@ -0,0 +1,31 @@
+from urllib import request
+from datetime import datetime
+import zipfile
+from pathlib import Path
+
+
+def main():
+    # assign the name of the file that will be stored in the data/ directory.
+    # current output: data/2020-05-22_fda_data.zip 
+    data_filename = f"{Path.cwd() / 'data' / str(datetime.date(datetime.now()))}_fda_data.zip"
```

Comment:
This line and subsequent lines fail if the `data` dir does not exist. We should either check to make sure this exists and warn the user it doesnt or have this directory created when this is run. In pathlib you can do this with the `mkdir` method  + flags `parents=True`, `exists_ok=True`

Reply:
I will merge and create an issue to be fixed.